build/routes/index.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.router = void 0;
+const express_1 = require("express");
+const router = (0, express_1.Router)();
+exports.router = router;
+//Basic ping/pong endpoint to test server functionality
+router.get('/ping', (req, res) => {
+    res.send('pong');
```

Comment:
Ideally, we would want to have a handler directory to hold route handler logic.  can you move this response func into something like a pingHandler and import that?